src/phoenix/server/api/mutations/dataset_mutations.py

```diff
@@ -154,7 +159,111 @@ async def add_spans_to_dataset(
                     for dataset_example_rowid, span in zip(dataset_example_rowids, spans)
                 ],
             )
-        return AddSpansToDatasetPayload(
+        return DatasetMutationPayload(
+            dataset=Dataset(
+                id_attr=dataset.id,
+                name=dataset.name,
+                description=dataset.description,
+                created_at=dataset.created_at,
+                updated_at=dataset.updated_at,
+                metadata=dataset.metadata_,
+            )
+        )
+
+    @strawberry.mutation
+    async def add_examples_to_dataset(
+        self, info: Info[Context, None], input: AddExamplesToDatasetInput
+    ) -> DatasetMutationPayload:
+        dataset_id = input.dataset_id
+        # Extract the span rowids from the input examples if they exit
+        span_ids = [
+            cast(GlobalID, example.span_id)
+            for example in input.examples
+            if example.span_id is not UNSET
+        ]
+        span_rowids = {
+            from_global_id_with_expected_type(global_id=span_id, expected_type_name=Span.__name__)
+            for span_id in set(span_ids)
+        }
+        dataset_version_description = (
+            input.dataset_version_description
+            if isinstance(input.dataset_version_description, str)
+            else None
+        )
+        dataset_version_metadata = input.dataset_version_metadata
+        dataset_rowid = from_global_id_with_expected_type(
+            global_id=dataset_id, expected_type_name=Dataset.__name__
+        )
+        async with info.context.db() as session:
+            if (
+                dataset := await session.scalar(
+                    select(models.Dataset).where(models.Dataset.id == dataset_rowid)
+                )
+            ) is None:
+                raise ValueError(
+                    f"Unknown dataset: {dataset_id}"
+                )  # todo: implement error types https://github.com/Arize-ai/phoenix/issues/3221
+            dataset_version_rowid = await session.scalar(
+                insert(models.DatasetVersion)
+                .values(
+                    dataset_id=dataset_rowid,
+                    description=dataset_version_description,
+                    metadata_=dataset_version_metadata or {},
+                )
+                .returning(models.DatasetVersion.id)
+            )
+            print("dataset_version_rowid" + str(dataset_version_rowid))
+            print("span_ids" + str(span_ids))
```

Comment:
Can we nix these print statement?

Reply:
Cool thing I recently learned, you can use `print(f"{dataset_version_rowid=}")` as a shorthand.

Reply:
Oh neat